src/base/bittorrent/session.cpp

```diff
@@ -346,6 +347,7 @@ Session::Session(QObject *parent)
     pack.set_int(libt::settings_pack::ssl_listen, 0);
     // To prevent ISPs from blocking seeding
     pack.set_bool(libt::settings_pack::lazy_bitfields, true);
+    pack.set_int(libt::settings_pack::alert_queue_size, 10000);
```

Comment:
Maybe it would be better to correlate this with the number of torrents in the resume queue? Say, `std::min(1000, numOfTorrents+100)`.